### PR TITLE
Some More

### PR DESCRIPTION
**Pull request by itsmat32143**
I think i have done this all correctly 😂

I didn't know what titles to put them under and where under the titles to put them you seem to have some sort of order to it so i just put them all at the top

Hope this helped you out a bit more than posting them in the issue page

### CODE REVIEW

File: pihole-google.txt [hunk 1]
```diff
@@ -1,6 +1,59 @@
...
+firebaseperusertopics-pa.clients6.google.com
+us-central1-events-d07ac.cloudfunctions.net
+redirector.gvt1.com
+inputtools.google.com
+appassets.androidplatform.net
+stun.l.google.com
+r3---sn-8pgbpohxqp5-ac5e.gvt1.com
+r4---sn-8pgbpohxqp5-ac5e.gvt1.com
+
+---------------------------------
```
Comment (nickspaargaren) [inline pihole-google.txt:55]: This line is very helpful but most likely invalid, can you remove this for me?

Comment (itsmat32143) [reply pihole-google.txt:55]: Which Lines are you referring to ? 
The ---- one ?

Comment (nickspaargaren) [reply pihole-google.txt:55]: Correct! Line 55, thanks!

**Comment by itsmat32143** [comment]
Haha no problem haven't done this before so didnt know all the stuff that i add turns green 😂thats why I put the line just to separate the stuff i added so yous could see haha 

What does # do before the titles to my knowledge ! Is what you put before titles so they aren't included in the list when scanned by adguard i assume pihole uses the same ??

**Comment by PoorPocketsMcNewHold** [comment]
> What does # do before the titles to my knowledge ! Is what you put before titles so they aren't included in the list when scanned by adguard i assume pihole uses the same ??

That's exact. Those are comments.

**Comment by PoorPocketsMcNewHold** [comment]
Gonna do that on the master branch, since i´ve just understood that i actually need to ¨push¨ the review, on top of actually writing the comments on a push request.

**Comment by nickspaargaren** [comment]
Okay you're right @PoorPocketsMcNewHold, way better organized. Thanks!

**Pull request merged**